frontend/src/__tests__/index.test.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { describe, expect, it, jest } from "@jest/globals";
+import AppRouter from "../app/AppRouter";
+
+describe("index.jsx file", () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+  it("should call ReactDOM.render() with <AppRouter/> component", () => {
+    jest.spyOn(ReactDOM, "render").mockImplementation(() => {});
+
+    const root = document.createElement("div");
+    root.id = "root";
+    document.body.appendChild(root);
+
```

Comment:
EsLint doesn't like require() however I don't have another way of importing the file (which runs the contents immediately) since imports need to be at the top of the file